Source/Dafny/Reporting.cs

```diff
@@ -68,6 +52,8 @@ public abstract class ErrorReporter {
       Message(source, ErrorLevel.Error, tok, msg);
     }
 
+    public abstract int Count(ErrorLevel level);
```

Comment:
I think it's worth a comment here that there's no guarantee that a reporter will track the number of messages.

Reply:
Isn't that guarantee part of the contract of `Count` ?

Reply:
I don't think it can be given that `LogErrorReporter` always returns `0`.

Reply:
It wasn't used. I've deleted it.

Reply:
Even better! Thanks! :)